decorators/decorators.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
```

Comment:
Suppose I have a Django project with a module named `decorators.py`, which is a common practice.
Within my `decorators.py` I want to import this (base) `Decorator` class, but a simple `from decorators import Decorator` would not work because of the name conflict. This situation can probably be solved using `absolute_import` (from `__future__`) in my module, but still I would prefer not to have any issue of this kind at all. Thus, why not choosing a better name for this package and module, something like `decorator_utils`?